package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "schemaindex",
+  "version": "1.0.0",
+  "description": "A public schema registry.",
+  "main": "index.js",
+  "directories": {
+    "test": "tests"
+  },
+  "prettier": {
+    "singleQuote": true,
+    "trailingComma": "es5"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "lint": "eslint .",
+    "check-formatting": "prettier \"core/static/**/*.{js,css}\" \"README.md\" --check",
+    "format": "prettier \"core/static/**/*.{js,css}\" \"README.md\" --write",
+    "typecheck": "tsc",
+    "prepare": "husky"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/dtinit/schemaindex.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
```

Comment:
Shouldn't this be consistent with our LICENSE file which uses Apache License, Version 2.0?

Reply:
Good catch, fixed this.